### PR TITLE
NCL-4820 Ensure project repositories aren't used with the old maven plugin

### PR DESCRIPTION
**Pull request by geoand**

### CODE REVIEW

File: manipulation/src/main/java/org/jboss/gm/manipulation/actions/MavenPublicationRepositoryAction.java [hunk 1]
```diff
@@ -46,9 +46,11 @@ public void execute(Project project) {
         if (uploadArchives == null) {
             project.getLogger().info("Creating uploadArchives task");
             uploadArchives = project.getTasks().create("uploadArchives", Upload.class);
+        } else {
+            uploadArchives.getRepositories().clear();
         }
 
-        Configuration config = ConfigCache.getOrCreate(Configuration.class);
+        Configuration config = ConfigCache.get(Configuration.class);
```
Comment (rnc) [inline manipulation/src/main/java/org/jboss/gm/manipulation/actions/MavenPublicationRepositoryAction.java]: Why is this changed to get?

Comment (geoand) [reply manipulation/src/main/java/org/jboss/gm/manipulation/actions/MavenPublicationRepositoryAction.java]: My mistake, I'll revert. I was trying various things and this got left behind

Comment (geoand) [reply manipulation/src/main/java/org/jboss/gm/manipulation/actions/MavenPublicationRepositoryAction.java]: fixed

**Comment by rnc** [comment]
Ok looks good!
